rlp/celo_raw.go

```diff
@@ -0,0 +1,58 @@
+package rlp
+
+import (
+	"encoding/binary"
+	"math"
+)
+
+// for payloads longer than 55 bytes for both strings and lists additional prefix signalizing
+// the total length of the payload is needed
+const payloadLengthThreshold = 0x37 // 55 in dec
+
+// indicates that the payload is a list
+const shortListEncodingByte = 0xC0
+
+// listEncodingByte (0xC0) + payloadLengthThreshold (0x37) = 0xF7
+const longListEncodingByte = 0xF7
+
+// Combine takes two RLP-encoded values one and two and produces a combined one
+// as if it was an RLP encoding of the [one, two] list
+// based on https://ethereum.org/en/developers/docs/data-structures-and-encoding/rlp/
+func Combine(one []byte, two []byte) []byte {
```

Comment:
Weird that there's no existing code for this. Could you add a simple test?

Reply:
Yeah, I was also surprised there is nothing like that. I'll add some.

Reply:
Added.